src/OrleansSQLUtils/Storage/AdoNetInvariants.cs

```diff
@@ -6,6 +9,18 @@ namespace Orleans.SqlUtils
     public static class AdoNetInvariants
     {
         /// <summary>
+        /// A list of the supported invariants.
+        /// </summary>
+        public static ICollection<string> Invariants { get; } = new Collection<string>(new List<string>(new[]
```

Comment:
Aknowledged. If allowed, I prioritize this lower if I start to run out of time. As this can be refactored on a second pass too (maybe some things on testing too, also feel free to propose specific kinds of tests if you think they're warranted).